build_platform.py

```diff
@@ -38,6 +38,8 @@
     "esp32" : "esp32:esp32:featheresp32:FlashFreq=80",
     # Adafruit AVR
     "trinket" : "adafruit:avr:trinket5",
```

Comment:
remove this?

Reply:
Sure; Didn't want to break anything that is using it but I can remove it